packages/listview/package.json

```diff
@@ -12,9 +12,13 @@
     },
     "dependencies": {
         "@rainbow-modules/app": "^0.48.0",
+        "@rainbow-modules/content": "^0.48.0",
         "@rainbow-modules/firebase-hooks": "^0.48.0",
+        "@rainbow-modules/hooks": "^0.48.0",
         "@rainbow-modules/icons": "^0.48.0",
         "@rainbow-modules/validation": "^0.48.0",
+        "firebase": "^8.2.0",
```

Comment:
since whatever project that uses this need to have installed firebase for define their own firebase config and pass it to the RainbowFirebaseApp component, then we should put firebase as a peer dep